### ISSUE TITLE
Disallow interactive content descendant in <summary>

### ISSUE THREAD

**Issue by zcorpan** [validator/validator#450]
See https://github.com/whatwg/html/pull/2256/commits/28590b2e50fa1be675f177c40abae305a640d34e

```html
<details>
 <summary><a href="x">x</a></summary>
</details>
```

**Comment by sideshowbarker** [comment]
Do we still want to do this? Or should the spec be changed to say it’s allowed? There do seem to be some reasonable cases—like what we have (had?) in the formatting of the WPT stability-checker notifications.

**Comment by zcorpan** [comment]
See https://github.com/whatwg/html/issues/2272#issuecomment-273926403 -- `a` should probably be allowed but unclear about other interactive content...
